### PR TITLE
:seedling: Bump CAPI to v1.8.2

### PR DESCRIPTION
**Pull request by mquhuy**
Bumps CAPI to v1.8.2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,9 +50,9 @@ need to be manually installed. Example flow of installing Metal3 provider:
    level of the logging verbose with a positive integer number, ie. -v5.
 
    ```shell
-   clusterctl init --core cluster-api:v1.8.1 \
-       --bootstrap kubeadm:v1.8.1 \
-       --control-plane kubeadm:v1.8.1 -v5
```
Comment (kashifest) [inline README.md:55]: I would rather keep this doc to point to 1.8.0 only and not update it in every patch from now on.

**Review by kashifest — COMMENTED** [review]
/approve

File: README.md (hunk 1 above)
Comment (tuminoid) [reply README.md:55]: Yup, put 1.8.0 back or untouch it here at least.

**Comment by adilGhaffarDev** [comment]
/hold
until comments are addressed

**Comment by mquhuy** [comment]
> /hold until comments are addressed

Would you want to point back to 1.8.0, or 1.8.1?

**Comment by mquhuy** [comment]
/unhold
README was put back to CAPI 1.8.0

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Comment by tuminoid** [comment]
> /cherry-pick release-1.8

Oh we have #1964 already, so close the cherry-pick when it is created.

**Pull request merged**